File: app/src/main/java/org/schulcloud/mobile/data/model/requestBodies/SignedUrlRequest.java
package org.schulcloud.mobile.data.model.requestBodies;

public class SignedUrlRequest {
    public final static String ACTION_OBJECT_GET = "getObject";
    public final static String ACTION_OBJECT_PUT = "putObject";

    public String action;
    public String path;
    public String fileType;

    public SignedUrlRequest(String action, String path, String fileType) {
        this.action = action;
        this.path = path;
        this.fileType = fileType;
    }
}

File: app/src/main/java/org/schulcloud/mobile/models/file/SignedUrlRequest.kt
package org.schulcloud.mobile.models.file


/**
 * Date: 7/5/2018
 */
class SignedUrlRequest {
    companion object {
        val ACTION_GET = "getObject"
        val ACTION_PUT = "putObject"
    }

    var parent: String? = null
    var filename: String? = null
    var fileType: String? = null
}
